healthcheck.js

```diff
@@ -0,0 +1,34 @@
+const mjml2html = require('mjml');
+
+const testInput = `<mjml>
+                        <mj-body>
+                            <mj-section>
+                                <mj-column>
+                                    <mj-text font-size="16px">Test</mj-text>
+                                </mj-column>
+                            </mj-section>
+                        </mj-body>
+                    </mjml>`
```

Comment:
really necessary having this code here?

Reply:
either that or loading it from a file, doesn't seem like the CLI or the lib has a "test" function